docs/sdks/client-sdks/ios/quickstart.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: Quickstart
+sidebar_position: 2
+---
+
+The Eppo iOS SDK enables feature flags and experiments in your iOS applications with just a few lines of code. It handles flag evaluation and experiment assignments locally on the device, making it fast and reliable for your users.
+
+## Installation
+
+While in XCode:
+
+1. Choose `Package Dependencies`
+2. Click `+` and enter package URL: `git@github.com:Eppo-exp/eppo-ios-sdk.git`
+3. Set dependency rule to `Up to Next Minor Version` and select `Add Package`
+4. Add to your project's target.
```

Comment:
Somewhere in the code examples please show how to import the package:

`import EppoFlagging`